linux.md

```diff
@@ -0,0 +1,40 @@
+# .NET for Linux Distributions
+
+.NET can be [installed on Linux](https://learn.microsoft.com/dotnet/core/install/linux) in multiple ways, via packages, [scripts](https://github.com/dotnet/install-scripts), or [tarballs](https://dotnet.microsoft.com/en-us/download). It can be installed globally, locally/privately, or with an app.
```

Comment:
```suggestion
.NET can be [installed on Linux](https://learn.microsoft.com/dotnet/core/install/linux) in multiple ways, via packages, [scripts](https://github.com/dotnet/install-scripts), or [tarballs](https://dotnet.microsoft.com/download?initial-os=linux). It can be installed globally, locally/privately, or with an app.
```